blog/_posts/2024-04-01-placeholder-post.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "New AKS Engineering Blog"
+description: "The new blog is ready!"
+date: 2024-04-01
+author: Brian Redmond
+categories: general
+---
+
+## Welcome to the new AKS Engineering Blog
+
+Full description here
```

Comment:
Are we supposed to have content here? At least an introduction? Or will this be hidden for now?

Reply:
These are just placeholders and will be removed when we add the first post. We need something there to render the site and test